cluster/recover.go

```diff
@@ -0,0 +1,40 @@
+package cluster
+
+import (
+	"fmt"
+
+	"github.com/canonical/go-dqlite"
+	dqliteClient "github.com/canonical/go-dqlite/client"
+)
+
+// LocalMember is the information that can be derived locally about a cluster member
+// without access to the dqlite database.
+type LocalMember struct {
```

Comment:
Should this be in a public package? I think this might fit better in the `internal/recover` or `internal/db` packages, no?

Reply:
`LocalMember` is exposed in `GetLocalClusterMembers` and `RecoverFromQuorumLoss` with the idea that consumers can modify it in ways other than just rendering Yaml (as discussed in our sync).